### ISSUE TITLE
`--xcodeindentation enabled` doesn't match Xcode for after closing parenthesis on its own line

### ISSUE THREAD

**Issue by jgale** [nicklockwood/SwiftFormat#1163]
This is what I get from `swiftformat . --xcodeindentation enabled` on 0.49.6:

```swift
    func execute3() -> AnyPublisher<TraceEvent, Never> {
        Publishers.CombineLatest(
            observePlaybackStartEvents(),
            isUserLoggedIn()
        )
            .compactMap {
                .map(playbackStartEvent: $0.0, isLoggedIn: $0.1)
            }
    }

    public var body: some View {
        TextField(
            "Hello, World",
            text: $name
        )
            .textContentType(.givenName)
            .padding()
    }
```

Xcode 13.3 itself will not indent those lines if they come directly after a closing parenthesis on its own line:

```swift
    func execute3() -> AnyPublisher<TraceEvent, Never> {
        Publishers.CombineLatest(
            observePlaybackStartEvents(),
            isUserLoggedIn()
        )
        .compactMap {
            .map(playbackStartEvent: $0.0, isLoggedIn: $0.1)
        }
    }

    public var body: some View {
        TextField(
            "Hello, World",
            text: $name
        )
        .textContentType(.givenName)
        .padding()
    }
```

Notes:
* This is similar/duplicate to https://github.com/nicklockwood/SwiftFormat/issues/1081, but it was unclear to me what action was being requested in that bug. IMO, it's definitely a bug that it doesn't match Xcode with `--xcodeindentation enabled`
* I believe this is a regression, as it didn't do this in `0.48.11`

**Comment by nicklockwood** [comment]
@jgale I can confirm that this behavior was explicitly added in 0.48.12, presumably to match the behavior of Xcode at the time. Since Xcode no longer behaves this way I'll update SwiftFormat to match.

**Comment by jgale** [comment]
Thank you! And thanks for all you do @nicklockwood.

**Comment by nicklockwood** [comment]
@jgale fixed in 0.49.7
